dissect/target/tools/shell.py

```diff
@@ -230,6 +236,27 @@ def do_man(self, line: str) -> bool:
     def complete_man(self, *args) -> list[str]:
         return cmd.Cmd.complete_help(self, *args)
 
+    def do_unalias(self, aliased: str) -> bool:
+        """delete runtime alias"""
+        if aliased in self._runtime_aliases:
+            del self._runtime_aliases[aliased]
```

Comment:
There is slightly different behavior when using alias with more than 3 arguments. 
I think we ignore them, but bash accepts them